### DISCUSSION TITLE
Windows 11 Error

### DISCUSSION THREAD

**Discussion by wisi-testpilot** [fcakyon/labelme2coco#24] [General]
Hi
I am using labelme2coco with pixellib.
It works fine with Windows10, but not with Windows11.
The tool creates a directory and then wants to write to a text file. Then Errno13 arises. I would be very happy about your help. best regards Wilhelm 

I have tried different versions.
pip3 install scikit-image==0.18.3
pip3 install numpy==1.19.5
pip3 install tensorflow==2.5.0
pip3 install imgaug
pip3 install pixellib --upgrade
pip3 install labelme2coco==0.1.2
![labelme2coco-Error](https://user-images.githubusercontent.com/91163307/229936039-0450f64b-ecc9-482c-9adc-dc84bedec117.jpg)

**Comment by wisi-testpilot** [comment]
see also this tread:

https://github.com/ayoolaolafenwa/PixelLib/issues/123

**Comment by fcakyon** [comment]
The error says you don't have any permission to write that file. You can try to give write permission to the program or change the directory to somewhere else where the program has write permission.

**Comment by wisi-testpilot** [comment]
Hi,
it has nothing to do with permissions. Only the very first version works with Windows 11. Please try it.

see also here:

https://github.com/ayoolaolafenwa/PixelLib/issues/123#issuecomment-1201339489

Best regards 
Wilhelm

**Reply by fcakyon** [reply]
Have you tried this solution? https://github.com/ayoolaolafenwa/PixelLib/issues/137#issuecomment-1118991111

**Comment by wisi-testpilot** [comment]
Yes, as I wrote, only the very first version (0.1.0) works with Windows 11.

The permission error is because Labelme2coco generates a folder and then wants to write to a text file.

Have a nice day,
best regards,
Wilhelm
